### DISCUSSION TITLE
Labels files for 10k dataset

### DISCUSSION THREAD

**Discussion by NiranjanRavi1993** [bdd100k/bdd100k#214] [General]
Hi,
    I am working with object detection using the bdd10k dataset. I was able to download the dataset but I could not find the labels for train and val. 

However, I could only find labels for bdd100k. It would be helpful if I can get labels for bdd10k?

Thanks and Regards
Niranjan Ravi.

**Comment by fyu** [comment]
You can check here about how to get the labels: https://doc.bdd100k.com/download.html#detection-2020-labels

**Reply by ShuchunXu** [reply]
> You can check here about how to get the labels: https://doc.bdd100k.com/download.html#detection-2020-labels

hello,this path is not avaliable,can you give me some suggestions?

**Comment by Ayakhaled200** [comment]
@NiranjanRavi1993 have you found them?

**Comment by Nimra28** [comment]
Can any one tell me the PC requirements to work on this dataset?

**Comment by ShuchunXu** [comment]
> Hi, I am working with object detection using the bdd10k dataset. I was able to download the dataset but I could not find the labels for train and val.
> 
> However, I could only find labels for bdd100k. It would be helpful if I can get labels for bdd10k?
> 
> Thanks and Regards Niranjan Ravi.

me too. do you solve this problem?
